### PR TITLE
Omniauth and user model

### PR DESCRIPTION
**Pull request by cdeliens**

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class SessionsController < ApplicationController
+  
+  def create
+  	user = User.sign_in_from_omniauth(env["omniauth.auth"])
```
Comment (cdeliens) [inline app/controllers/sessions_controller.rb]: pls fix identation

File: app/models/article.rb [hunk 2]
```diff
@@ -1,6 +1,19 @@
...
+#
+
 class Article < ActiveRecord::Base
   has_many :comments, dependent: :destroy
+  belongs_to :user
 
-  validates :title, presence: true, length: { minimum: 5 }
-  validates :text, presence: true, length: { maximum: 100 }
+  validates :title, presence: true, length: { in: 2..80 }
+  validates :text, presence: true, length: { maximum: 5000 }
```
Comment (cdeliens) [inline app/models/article.rb:18]: Cual es la razon para limitar el tamano del post?

File: app/models/comment.rb [hunk 3]
```diff
@@ -1,3 +1,16 @@
...
+#  id         :integer          not null, primary key
+#  body       :text
+#  article_id :integer
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+
 class Comment < ActiveRecord::Base
   belongs_to :article
+  validates :commenter, presence: true
```
Comment (cdeliens) [inline app/models/comment.rb]: creo que commenter no existe, como para validarlo

File: app/controllers/sessions_controller.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
+class SessionsController < ApplicationController
+  
+  def create
```
Comment (cdeliens) [inline app/controllers/sessions_controller.rb:3]: creo que es importante agregarle testing a este controller.

File: app/controllers/sessions_controller.rb (hunk 1 above)
Comment (JulioOrdonezV) [inline app/controllers/sessions_controller.rb]: no issues on my side

File: app/models/comment.rb (hunk 3 above)
Comment (JulioOrdonezV) [inline app/models/comment.rb]: Fixed it

File: app/models/article.rb (hunk 2 above)
Comment (JulioOrdonezV) [inline app/models/article.rb:18]: So they don't fill the database with random junk

File: app/controllers/sessions_controller.rb (hunk 4 above)
Comment (JulioOrdonezV) [inline app/controllers/sessions_controller.rb:3]: Ok, WIP

File: app/controllers/sessions_controller.rb (hunk 1 above)
Comment (JulioOrdonezV) [inline app/controllers/sessions_controller.rb]: Fixed it in lattest commit

File: app/controllers/sessions_controller.rb (hunk 4 above)
Comment (JulioOrdonezV) [inline app/controllers/sessions_controller.rb:3]: Added tests for the sessions controller. Please review them and provide feedback.

File: spec/factories.rb [hunk 5]
```diff
@@ -0,0 +1,10 @@
+require 'faker'
+
+FactoryGirl.define do
+  factory :user do
+    name "Example User"
```
Comment (cdeliens) [inline spec/factories.rb:5]: require 'faker' but not using it?

Comment (JulioOrdonezV) [inline spec/factories.rb:5]: Removed it on latest commit

**Comment by cdeliens** [comment]
:+1:

**Pull request merged**